addons/udes_transport_management/models/transport.py

```diff
@@ -61,7 +61,8 @@ def get_info(self, **kwargs):
     def prepare_transport_info(self, **kwargs):
         """Prepare transport info method that can be override in order
         to be able to extend it in other modules"""
-        self.ensure_one()
+        # Not calling self.ensure_one() in case the record does not exist which
```

Comment:
For now it makes sense the comment, but in future i dont think that will be needed.

Suggestion: Remove the comment and:
`result = []`
`for transport in self:`
    `transport_info = {
                "label": "Trailer Number",
                "value": transport.vehicle_sequence or None,
                "lookup_key": "trailer_number",
            },
            {
                "label": "Trailer Unit ID",
                "value": transport.vehicle_description or None,
                "lookup_key": "trailer_unit_id",
            },
            {
                "label": "Vehicle Registration",
                "value": transport.license_plate or None,
                "lookup_key": "vehicle_registration",
            },
            {
                "label": "Driver Name",
                "value": transport.driver_name or None,
                "lookup_key": "driver_name",
            }`
    `result.append(transport_info)`
`return result`

Reply:
I wrote it so that in the future we won't forget and add it back again for any reason. Would it be better removing it you think?

Reply:
I addressed your suggestion in the code. I could not simply copy your code as it wouldn't return default values when no transport record is available.